recipes/gstreamer/all/conanfile.py

```diff
@@ -47,18 +47,18 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("glib/2.76.2", transitive_headers=True, transitive_libs=True)
+        self.requires("glib/2.76.3", transitive_headers=True, transitive_libs=True)
 
     def validate(self):
         if not self.dependencies.direct_host["glib"].options.shared and self.info.options.shared:
             # https://gitlab.freedesktop.org/gstreamer/gst-build/-/issues/133
             raise ConanInvalidConfiguration("shared GStreamer cannot link to static GLib")
 
     def build_requirements(self):
-        self.tool_requires("meson/1.1.0")
+        self.tool_requires("meson/1.1.1")
         # There used to be an issue with glib being shared by default but its dependencies being static
         # No longer the case, but see: https://github.com/conan-io/conan-center-index/pull/13400#issuecomment-1551565573 for context
-        self.tool_requires("glib/2.76.2")
+        self.tool_requires("glib/2.76.3")
```

Comment:
@RubenRBS leaving this at your criteria, but it may be a good opportunity to use `glib/<host_version>` - otherwise PR LGTM.